easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch

```diff
@@ -0,0 +1,42 @@
+diff -ru mmcif_pdbx-2.0.0/MANIFEST.in mmcif_pdbx-2.0.0_patched/MANIFEST.in
```

Comment:
A short description of what the patch does, reference to issue, and author are appreciated for patches